packages/core/src/common/abstractComponent.ts

```diff
@@ -12,6 +12,11 @@ import * as React from "react";
  * in order to add some common functionality like runtime props validation.
  */
 export abstract class AbstractComponent<P, S> extends React.Component<P, S> {
+    public displayName: string;
+
+    // Not bothering to remove entries when their timeouts finish because clearing finished timeout is no-op
+    private timeoutHandles: number[] = [];
```

Comment:
i chose `handle` cuz that's what used in `lib.d.ts`. the parameter is called handle and `clearTimeout(handle: number)`.